xbmc/guilib/StereoscopicsManager.cpp

```diff
@@ -241,9 +243,28 @@ RENDER_STEREO_MODE CStereoscopicsManager::GetStereoModeOfPlayingVideo(void)
 
 const std::string &CStereoscopicsManager::GetLabelForStereoMode(const RENDER_STEREO_MODE &mode) const
 {
-  if (mode == RENDER_STEREO_MODE_AUTO)
-    return g_localizeStrings.Get(36532);
-  return g_localizeStrings.Get(36502 + mode);
+  int msgId;
+  switch(mode) {
```

Comment:
I'm not sure I like this construct here. Having only one special case was fine, but having half the labels hardcoded while the rest is "dynamic" seems a bit confusing. Maybe we should have a label mapping struct for all modes to streamline this a bit? @fritsch, opinions?